ymcatwincities/openy
# Easy 301 Redirects from Content Editing Flow

https://trello.com/c/D5wBxuBA/107-easy-301-redirects-from-content-editing-flow-score-6

Steps to review:
1. Open node edit page. Find Redirects fieldset on the right side. 
2. Add all URLs to redirect to this node and save it.
3. Go to Redirect list page "/admin/config/search/redirect?order=created&sort=desc" and check new redirects.
